### ISSUE TITLE
Python 3.9 support (regenerate with new cython)

### ISSUE THREAD

**Issue by unrelentingtech** [scikit-sparse/scikit-sparse#62]
The currently released 0.4.4 distribution on PyPI contains a `cholmod.c` generated with an older version of Cython that didn't support Python 3.9, so the build fails:

```
sksparse/cholmod.c:19722:40: error: no member named 'tp_print' in 'struct _typeobject'
  __pyx_type_8sksparse_7cholmod_Common.tp_print = 0;
  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ ^
sksparse/cholmod.c:19731:40: error: no member named 'tp_print' in 'struct _typeobject'
  __pyx_type_8sksparse_7cholmod_Factor.tp_print = 0;
  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ ^
sksparse/cholmod.c:19751:58: error: no member named 'tp_print' in 'struct _typeobject'
  __pyx_type_8sksparse_7cholmod__CholmodSparseDestructor.tp_print = 0;
  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ ^
sksparse/cholmod.c:19762:57: error: no member named 'tp_print' in 'struct _typeobject'
  __pyx_type_8sksparse_7cholmod__CholmodDenseDestructor.tp_print = 0;
  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ ^
```

Please release an update regenerated with newer Cython.

**Comment by stuart-nolan** [comment]
The following install method (Ubuntu 20.04.2 LTS, Python 3.9.1 in a virtual env, Cython 0.29.21, and suitesparse 5.8.1) works around this issue for me:
```
git clone https://github.com/scikit-sparse/scikit-sparse.git
cd scikit-sparse
python setup.py build_ext --library-dirs="/path/to/suitesparse-5.8.1/lib" --include-dirs="/path/to/suitesparse-5.8.1/include" install
nosetests -v sksparse

sksparse.test_cholmod.test_cholesky_smoke_test ... /home/ul/.pyenv/versions/3.9.1/envs/p391.mkl/lib/python3.9/site-packages/scikit_sparse-0.4.4-py3.9-linux-x86_64.egg/sksparse/test_cholmod.py:57: CholmodTypeConversionWarning: converting matrix of class dia_matrix to CSC format
  f = cholesky(sparse.eye(10, 10))
...
ksparse.test_cholmod.test_CholmodNotPositiveDefiniteError ... ok
sksparse.test_cholmod.test_natural_ordering_method ... ok

----------------------------------------------------------------------
Ran 10 tests in 9.443s # old laptop

OK
```

**Comment by jellis18** [comment]
This is now fixed in master and v0.4.5 which is available on pypi and conda-forge
